packages/block-editor/src/store/selectors.js

```diff
@@ -1036,7 +1036,57 @@ export function isTyping( state ) {
  * @return {boolean} Whether user is dragging blocks.
  */
 export function isDraggingBlocks( state ) {
-	return state.isDraggingBlocks;
+	return !! state.draggedBlocks;
+}
+
+/**
+ * Returns the client ids of any blocks being directly dragged.
+ *
+ * This does not include children of a parent being dragged.
+ *
+ * @param {Object} state Global application state.
+ *
+ * @return {string[]} Array of dragged block client ids.
```

Comment:
It could return `null` if not dragging blocks. Should we return `[]` or `null` in this case? :🤔. I feel like `[]` would be simpler to indicate that there's no blocks being dragged, and unify the type. We just have to check the `length` attribute in `isDraggingBlocks`.

Reply:
Yep, makes sense, fixed in 680714b  👍